listeners/websocket.js

```diff
@@ -0,0 +1,64 @@
+var WebSocketClient = require('websocket').client;
+var deviceInfo = require('./../data/device'),
```

Comment:
Is this path valid if the data folder has been moved? (iotkit-admin set-data-directory ...)